core/templates/services/math-interactions.service.ts

```diff
@@ -171,9 +181,38 @@ export class MathInteractionsService {
       this.warningText = 'Your answer contains an invalid character: "_".';
       return false;
     }
-    if (expressionString.match(/(\+$)|(\+\))/g)) {
+    if (expressionString.match(/^(\+$)|^(\(\+\))/g)) {
+      this.warningText = (
+        'Your answer seems to be missing a number before' +
+        ' the + operator.');
+      return false;
+    }
+
+    if (expressionString.match(/^(\*$)|^(\(\*\))|^(\(\*\d+\))|^(\*\d+)/g)) {
+      this.warningText = (
+        'Your answer seems to be missing a number before' +
+        ' the × operator.');
+      return false;
+    }
```

Comment:
This is the regex when user give input as `*, (*), *9 & (*9)` and update the error message as shown. 
Initially error message was `Not a prefix operator`, which was actual issue of this PR.